core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -144,6 +169,11 @@ var ExplorationEditorTranslationTab = function() {
       element(by.cssContainingText('option', language)).click();
   };
 
+  this.audioElem = function() {
```

Comment:
I don't understand what element this function is trying to query for.

Reply:
@nithusha21 This function is not needed. Directly storing element in audioElem.